src/main/scala/org/broadinstitute/dsde/firecloud/core/ExportEntitiesByType.scala

```diff
@@ -55,13 +55,13 @@ class ExportEntitiesByTypeActor(requestContext: RequestContext) extends Actor wi
           allSucceeded match {
             case true =>
               val entities = responses.flatMap(unmarshal[List[EntityWithType]].apply)
-              entityType match {
-                case x if x.endsWith("_set") =>
-                  val rootType = x.replace("_set", "")
-                  val zippedFileName = x + ".zip"
-                  val rootData = TSVFormatter.makeEntityTsvString(entities, rootType)
-                  val membershipData = TSVFormatter.makeMembershipTsvString(entities, entityType)
-                  val zipBytes: Array[Byte] = getZipBytes(filename, membershipData, rootType, rootData)
+              ModelSchema.getCollectionMemberType(entityType) match {
+                case Success(collectionType) if collectionType.isDefined =>
```

Comment:
Is it valid to fall back to `case _` if Failure() or `!collectionType.isDefined`?